components/Cards/tests/snapshot/Dummy.swift

```diff
@@ -0,0 +1,4 @@
+
+// Dummy swift file to ensure Snapshot tests compile
```

Comment:
Will be helpful for future folks to include in the description here the error that this file is resolving. Add the error message + any links you found that helped guide you to this solution please.

Reply:
Done